helpers/class.TestRunnerUtils.php

```diff
@@ -840,8 +860,8 @@ static private function getNavigatorMap(AssessmentTestSession $session, $itemInd
         $useTitle = !empty($config['test-taker-review-use-title']);
         $language = \common_session_SessionManager::getSession()->getInterfaceLanguage();
```

Comment:
Add this to use section:

`common_session_SessionManager`

```suggestion
        $language = common_session_SessionManager::getSession()->getInterfaceLanguage();
```